### PR TITLE
Added doc about rand-seed and rand-address

### PR DESCRIPTION
**Pull request by enricopro**
# Description of change

I added a small documentation about the tools `rand-seed` and `rand-address`. This came up to my mind since when I used GoShimmer for the first time I wasn't unable to find a tool to generate seeds, so I tried to build it through python, but a mod on Discord suggested me these two tools. I thought that just seeing them in the documentation would have saved me some time.

## Type of change

- Documentation Fix

## How the change has been tested

I did a grammar check of the added text.

## Change checklist

Add an `x` to the boxes that are relevant to your changes, and delete any items that are not.

- [x] My code follows the contribution guidelines for this project

### CODE REVIEW

File: documentation/docs/tooling/overview.md [hunk 1]
```diff
@@ -20,4 +20,5 @@ We provide a documentation for the following tools:
 - The [docker private network](docker_private_network.md) with which a local test network can be set up locally with docker.
 - The [integration tests](integration_tests.md) spins up a `tester` container within which every test can specify its own GoShimmer network with Docker.
 - The [cli-wallet](../tutorials/wallet_library.md) is described as part of the tutorial section.
-- The [DAGs Visualizer](dags_visualizer.md) is the all-round tool for visualizing DAGs.
\ No newline at end of file
+- The [DAGs Visualizer](dags_visualizer.md) is the all-round tool for visualizing DAGs.
+- The [rand-seed and rand-address](rand_seed_and_random_address.md) are used to randomly generate a seed, with the relative public key, or a random address.
```
Comment (lucas-tortora) [inline documentation/docs/tooling/overview.md]:
```suggestion
- The [rand-seed and rand-address](rand_seed_and_rand_address.md) to randomly generate a seed, with the relative public key, or a random address.
```

File: documentation/docs/tooling/rand_seed_and_rand_address.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
+---
+description: Running these tools let you generate random seeds and addresses through a simple command.
```
Comment (lucas-tortora) [inline documentation/docs/tooling/rand_seed_and_rand_address.md]:
```suggestion
description: 'You can use the rand-address and rand-seed tools to generate random seeds and addresses through a simple command.'
```

File: documentation/docs/tooling/rand_seed_and_rand_address.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+keywords:
+- address
+- seed
+- public key
+- private key
+- generate
+- generation
+---
+# Rand seed and rand address
```
Comment (lucas-tortora) [inline documentation/docs/tooling/rand_seed_and_rand_address.md]:
```suggestion
# Rand Seed and Rand Address

You can use the [`rand-address`](#rand-address) and [`rand-seed`](#rand-seed) tools to generate addresses and seeds in a single command.
```

File: documentation/docs/tooling/rand_seed_and_rand_address.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+## Rand address
+
+Rand address is a tool realized through a Go script that let you generate a random address. Its execution is pretty simple:
```
Comment (lucas-tortora) [inline documentation/docs/tooling/rand_seed_and_rand_address.md]:
```suggestion
## Rand Address

You can use the `rand-address` tool to generate a random address by running the following command from the `tools/rand-address` directory:
```

File: documentation/docs/tooling/rand_seed_and_rand_address.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+
+```shell
+cd tools/rand-address
+go run main.go
+13n6HnqiLQVaE2sp8BExM51C2z1BLw7SrFjNAUK439YCC
+```
+
+Right after the execution, you will have a new randomly generated address.
+
+## Rand seed
```
Comment (lucas-tortora) [inline documentation/docs/tooling/rand_seed_and_rand_address.md]:
```suggestion
## Rand Seed
```

File: documentation/docs/tooling/rand_seed_and_rand_address.md (hunk 3 above)
Comment (lucas-tortora) [inline documentation/docs/tooling/rand_seed_and_rand_address.md]:
```suggestion
# Rand Seed and Rand Address

You can use the [`rand-address`](#rand-address) and [`rand-seed`](#rand-seed) tools to generate addresses and seeds in a single command. 

You can later use the generated data to, for example, set up the exact same nodes in a private network.
```

File: documentation/docs/tooling/rand_seed_and_rand_address.md (hunk 4 above)
Comment (lucas-tortora) [inline documentation/docs/tooling/rand_seed_and_rand_address.md]:
```suggestion
You can use the `rand-address` tool to generate a random address by running the following command from the `tools/rand-address` directory:
```

File: documentation/docs/tooling/rand_seed_and_rand_address.md [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+
+Rand address is a tool realized through a Go script that let you generate a random address. Its execution is pretty simple:
+
+```shell
+cd tools/rand-address
+go run main.go
+13n6HnqiLQVaE2sp8BExM51C2z1BLw7SrFjNAUK439YCC
+```
+
+Right after the execution, you will have a new randomly generated address.
```
Comment (lucas-tortora) [inline documentation/docs/tooling/rand_seed_and_rand_address.md]:
````suggestion
```shell
cd tools/rand-address
go run main.go
```

### Expected Output

The script will output a Base58 string representing the newly generated address, for example:

```shell
13n6HnqiLQVaE2sp8BExM51C2z1BLw7SrFjNAUK439YCC
```
````

File: documentation/docs/tooling/rand_seed_and_rand_address.md [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+
+## Rand seed
+
+The rand seed tool is a CLI script a bit more complete than the previous one. It let you generate in a text file a seed (encoded in base58 and base64), its relative identity and its public key. The execution is still simple:
```
Comment (lucas-tortora) [inline documentation/docs/tooling/rand_seed_and_rand_address.md]:
```suggestion
## Rand Seed

You can use the `rand-address` tool to generate a text file with the following:

* A [seed](/../tutorials/send_transaction#seed), represented in Base64 and Base58.
* The seed's relative identity, as a Base58 string. 
* The relative identity's public, key in Base58.
```

File: documentation/docs/tooling/rand_seed_and_rand_address.md [hunk 8]
```diff
@@ -0,0 +1,44 @@
...
+
+```shell
+base64:ri9C8oAT3IPsus2j+IllMbW2B3nOqe4uC56zfr344zY=
+base58:CiwjnjMRwEbCGiATWjNsrVptBTNH13AHrVNmG31KK9cy
+Identity - base58:BCUnRc6c
+Identity - base58:BCUnRc6cv4YVnB3Rw5DDfdsFuVVUW97MyLEBzWxHqfQj
+Public Key - base58:Ht9VR8qAgmruDPzsQbak3AJvXcJY6q6Mxyaz4pDicDEw
+```
+
+These tools are useful if you have to run, for example, a Docker Private Network and you want to setup the exact nodes running in the network.
```
Comment (lucas-tortora) [inline documentation/docs/tooling/rand_seed_and_rand_address.md]:
````suggestion
### Expected Output

The script will generate a `random-seed.txt` file in the current working directory, for example:

```plaintext
base64:ri9C8oAT3IPsus2j+IllMbW2B3nOqe4uC56zfr344zY=
base58:CiwjnjMRwEbCGiATWjNsrVptBTNH13AHrVNmG31KK9cy
Identity - base58:BCUnRc6c
Identity - base58:BCUnRc6cv4YVnB3Rw5DDfdsFuVVUW97MyLEBzWxHqfQj
Public Key - base58:Ht9VR8qAgmruDPzsQbak3AJvXcJY6q6Mxyaz4pDicDEw
```
````

**Review by lucas-tortora — COMMENTED** [review]
Awesome article, thanks for contributing! Just a few comments regarding structure :) 

You will also need to alter the [sidebar.js file](https://github.com/iotaledger/goshimmer/blob/develop/documentation/sidebars.js#L266)  to add an entry for this article :)

**Comment by enricopro** [comment]
@lucas-tortora thank you a lot for your feedback. Unfortunately, I'm not expert in contributions, so please have patience for my mistakes. Anyway, I have committed your suggestions and updated sidebar.js. Thank you again for your time! :)

File: documentation/docs/tooling/rand_seed_and_rand_address.md [hunk 9]
```diff
@@ -0,0 +1,57 @@
...
+
+### Expected Output
+
+The script will output a Base58 string representing the newly generated address, for example:
+
+```shell
+13n6HnqiLQVaE2sp8BExM51C2z1BLw7SrFjNAUK439YCC
+```
+
+## Rand Seed
```
Comment (lucas-tortora) [inline documentation/docs/tooling/rand_seed_and_rand_address.md:32]:
```suggestion
```

Sorry, it seems Ieft a double header here while suggesting.

**Review by lucas-tortora — APPROVED** [review]
It seems I left a doubleheader while suggesting, sorry about that.  Other than that, LGTM! 

Thanks again for contributing :)

**Review by piotrm50 — APPROVED** [review]

File: documentation/docs/tooling/rand_seed_and_rand_address.md [hunk 10]
```diff
@@ -0,0 +1,55 @@
...
+
+You can use the `rand-address` tool to generate a text file with the following:
+
+* A [seed](/../tutorials/send_transaction#seed), represented in Base64 and Base58.
```
Comment (piotrm50) [inline documentation/docs/tooling/rand_seed_and_rand_address.md]:
```suggestion
* A [seed](../tutorials/send_transaction#seed), represented in Base64 and Base58.
```

Comment (piotrm50) [reply documentation/docs/tooling/rand_seed_and_rand_address.md]: Link needs to point to relative instead of absolute path.

**Pull request merged**